<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderBlockClass</class>
 <widget class="QWidget" name="OrderBlockClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>45</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>2</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OrderBlock</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLineEdit" name="LE_Order">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 74px;
height: 21px;
background: #F3F7FD;
border: 1px solid #92A8C6;
border-radius: 2px;

font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #1A1A1A;
line-height: 60px;</string>
     </property>
     <property name="text">
      <string>ping 127.0.0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_Send">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 120px;
height: 28px;
background: #31B880;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_Delete">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 120px;
height: 28px;
background: #E0463E;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>4</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid 
rgb(0, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="PB_Yes">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">width: 120px;
height: 28px;
background-color: rgb(255, 149, 186);
border-radius: 4px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_No">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">width: 120px;
height: 28px;
background-color: rgb(170, 0, 127);
border-radius: 4px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
